core/pkg/runtime/from_config_test.go

```diff
@@ -34,55 +34,55 @@ func TestParseSource(t *testing.T) {
 			out: []SourceConfig{
 				{
 					URI:      "config/samples/example_flags.json",
-					Provider: "file",
+					Provider: syncProviderFile,
 				},
 				{
 					URI:         "http://test.com",
-					Provider:    "http",
+					Provider:    syncProviderHTTP,
 					BearerToken: ":)",
 				},
```

Comment:
maybe worth adding another entry to the in/out array with `https` and `grpcs`

Reply:
Resolving after adding required tests :)